interface/main/calendar/modules/PostCalendar/pntemplates/default/views/month/ajax_template.html

```diff
@@ -538,10 +538,10 @@
                 else {
                 //Category Clinic closaed or holiday take the event title
                      if ( $catid == 6 || $catid == 7){
-                        $content  = $event['title'];
+                        $content  = xlt($event['title']);
                     }else{
                     // no patient id, just output the category name
-                        $content .= $catname;
+                        $content .= xlt($catname);
```

Comment:
There is a specific function for translating the appt category names:
https://github.com/openemr/openemr/blob/master/library/translation.inc.php#L186
``````
$content .= text(xl_appt_category($catname));
``````